CIP-1775/README.md

```diff
@@ -0,0 +1,108 @@
+---
+CIP: 1775
+Title: Delegated Spending Authorizations
+Authors: Ivan Mikushin <sorrel_cantos.0@icloud.com>
+Comments-URI:
+Status: Proposed
+Type: Standards Track
+Created: 2022-11-17
+License: CC-BY-4.0
+---
+
+# Delegated Spending Authorizations
+
+"Give me custody, or give me death!" - [@ArmySpies](https://twitter.com/ArmySpies/status/1592586212816355328)
+
+## Abstract
+
+We propose to allow users to delegate authorized spending of outputs in their wallets to smart contracts or other users, without sending said outputs out of their wallets.
+
+## Motivation
+
+In order to use any dApp on Cardano (e.g. a decentralized exchange or an NFT marketplace), a user has to **deposit tokens** to a script address, thereby **giving up custody** of the tokens.
+
+This can be problematic for a number of reasons.
+
+- The user loses control of the assets they send out to a smart-contract, unless the smart contract is specifically designed to allow for such control (e.g. giving a "Cancel trade" or "De-list NFT" button in the UI).
```

Comment:
I'm confused about why this is a bad thing, and I think that's a crux of disagreement.

At the moment, an output which has an address A is _exclusively_ spendable according to that address. This is a useful property! Consider e.g. the fact that many dapps rely on collateral or deposits of some kind - most of these systems _do not work_ if the depositor can simply reclaim their collateral by using an alternative spending condition.

As you point out, today you can _deliberately_ remove exclusivity by building it into your script.

IMO, the main effect of this CIP would be to _flip the default_: addresses would by default become non-exclusive. But as we saw, exclusivity is _important_ to the functioning of many applications. So you would need a way to ensure you _did_ have exclusivity, which seems both tricky and an annoying footgun for application developers (didn't add the check to ensure exclusivity? Oops, users went off with all their deposits).

So:
- Today: exclusivity is default and useful, applications can build in non-exclusivity easily
- With this CIP: non-exclusivity is default, applications can build in exclusivity (?) but it's harder

This doesn't look like a good trade to me.

Reply:
Okay, I hadn't quite realised that the addition is asymmetrical. So instead we'd have a situation where applications will have to decide whether to do anything with outputs that have delegated spending conditions or not. That is indeed less unsafe although weirdly asymmetrical, and I need to think about it.

Reply:
I'm updating the text so that only beginning with Plutus v3, scripts will be able to participate in DSAs or receive outputs with DSAs. This will restore the symmetry.

The version bump is necessary in order to preserve the exclusivity assumptions of existing scripts and to be able to pass DSAs via script context.